.github/pull_request_template.md

```diff
@@ -10,5 +10,5 @@
 - [ ] Jest frontend tests
 - [ ] Cypress end-to-end tests
 - [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
-- [ ] Frontend/CSS is usable at 320px (iPhone SE) and decent at 360px (most phones)
-- [ ] Breaking changes are backwards-compatible. Ensure old/new frontend requests work with new/old backends, and vice versa.
+- [ ] New/changed UI looks good on mobile (screen width 360 px)
```

Comment:
This changes the bullet. First, we must still look at 320px (iPhone mini/SE). Second, for the moment, we're going for "decent", not "good". Not yet.

The main thing I want now is that we don't hardcode values like "width: 400px" into the interface, and just check once that the content flows fine on a smaller resolution. Making it "good", which I read as "looks like a good mobile app", is misdirected effort right now.

Reply:
Hm, I re-reworded it like so: `New/changed UI is decent on smartphones (viewport width around 360px)`.
Keeping it a bit more compact than before + highlighting the need to check UI affected by the PR.
(BTW the current iPhone SE has a 375px-wide screen 🤔)